src/public/media.ts

```diff
@@ -451,18 +560,32 @@ export namespace media {
     }
 
     const params = [mediaInputs];
-    // What comes back from native at attachments would just be objects and will be missing getMedia method on them.
-    sendMessageToParent('selectMedia', params, (err: SdkError, localAttachments: Media[]) => {
-      if (!localAttachments) {
-        callback(err, null);
-        return;
-      }
-      const mediaArray: Media[] = [];
-      for (const attachment of localAttachments) {
-        mediaArray.push(new Media(attachment));
-      }
-      callback(err, mediaArray);
-    });
+    // What comes back from native as attachments would just be objects and will be missing getMedia method on them
+    sendMessageToParent(
+      'selectMedia',
```

Comment:
This name should always include the namespace/module so in this case it should be `media.selectMedia` and not just `selectMedia`

Reply:
As discussed on the call, will take all these changes in the next clean-up PR. 
Task 2096888: [JS SDK] Clean all the API signatures to have the namespace prefix